server/adapters/restore.js

```diff
@@ -0,0 +1,49 @@
+//
+// Adapter Restore
+//
+module.exports = function initRestoreAdapters(models, adapters) {
+
+  //
+  // Create new restore instance
+  //
+  adapters.createRestore = function (userId, saveId) {
```

Comment:
@BernardJeremy @clairealmozinos 
From what I remember that we have established during the architecture design meetups, the `adapters` should contain entites similar to the one recognizable as `models` and as `workers`. Here you just use it as a table of functions.